# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/none/MacroAssembler-none.h
# Commit: 69b8eca9b1ef
# Full Hash: 69b8eca9b1ef2afb591cf9cf72799cbf16dc16bb
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-20 14:55:47
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 1: Reserve two slots after stack arguments for the future tls preservation. r=lth
#   
#   We are going to remove Frame::tls and support trampolines for indirect calls, so we need to get rid of using Frame::tls.
#   In this and the followup patches I will iteratively remove all dependencies of Frame::tls and remove it eventually.
#   
# ==============================================================================

diff -r de127b4ae931 -r 69b8eca9b1ef js/src/jit/none/MacroAssembler-none.h
--- a/js/src/jit/none/MacroAssembler-none.h	Tue Oct 20 10:26:41 2020 +0000
+++ b/js/src/jit/none/MacroAssembler-none.h	Tue Oct 20 10:24:07 2020 +0000
@@ -619,6 +619,7 @@
   ABIArg next(MIRType) { MOZ_CRASH(); }
   ABIArg& current() { MOZ_CRASH(); }
   uint32_t stackBytesConsumedSoFar() const { MOZ_CRASH(); }
+  void increaseStackOffset(uint32_t) { MOZ_CRASH(); }
 };
 
 static inline bool GetTempRegForIntArg(uint32_t, uint32_t, Register*) {